tests/test_cases/issue_2362/issue_2362.py

```diff
@@ -0,0 +1,14 @@
+"""
+Test that cocotb doesn't die if a new logging level is added
+"""
+import cocotb
+import logging
+
+
+@cocotb.test()
+async def test_alu(dut):
```

Comment:
Perhaps a more descriptive test name?
```suggestion
async def test_custom_log_level(dut):
```